src/__tests__/generateConfluenceMarkdown.test.ts

```diff
@@ -24,24 +26,23 @@ const fakeIssues: Issue[] = [
 describe('generateConfluenceMarkdown function', () => {
     it('should pass in a dependency with no issues', () => {
         const mapWithoutIssues: Map<string, Issue[]> = new Map();
-        // const noIssues: Issue[] = [];
+        const noIssues: Issue[] = [];
         const dependency = 'TestDependency';
-        const oneDependency = mapWithoutIssues.set(dependency, fakeIssues);
-
+        const oneDependency = mapWithoutIssues.set(dependency, noIssues);
         const results = generateConfluenceMarkdown(oneDependency, 30);
-        expect(results).toContain([]); // expecting it to fail because fakeIssues has two issues...
+
+        expect(results).toContain(noIssues);
     });
 
-    // Tests WIP
-    // it('should return 2 issues for a single dependency', () => {
-    //     const mockDependencyMap: Map<string, Issue[]> = new Map();
-    //     const dependency2 = 'NodeJsDependency';
-    //     const twoIssues = mockDependencyMap.set(dependency2, fakeIssues);
-    //     const dep = mockDependencyMap.get(dependency2[1]);
+    it('should return 2 issues for a single dependency', () => {
```

Comment:
Several of these "it" messages should be changed. Let's get on slack to run through them all together.